lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/QtiImport.java

```diff
@@ -1128,7 +1157,11 @@ boolean procshort(Node item) throws IOException{
 	    if (ident == null)
 		ident = ans.qident;
 
-	    out.println("<response_str ident=\""+rident+"\" rcardinality=\"Ordered\" rtiming=\"No\"><render_fib charset=\"ascii-us\" columns=\"20\" encoding=\"UTF_8\" fibtype=\"String\" prompt=\"Box\" rows=\"1\"></render_fib>");
+	    if (type.equalsIgnoreCase("Fill In the Blank")) {
+		    out.println("<response_str ident=\""+rident+"\" rcardinality=\"Ordered\" rtiming=\"No\"><render_fib charset=\"ascii-us\" columns=\"20\" encoding=\"UTF_8\" fibtype=\"String\" prompt=\"Box\" rows=\"1\"></render_fib>");
```

Comment:
My preference is to use spaces around + signs in concatenation, and this code uses that approach elsewhere. It just makes it easier to read.

Reply:
Solved